tests/store/model_registry/test_file_store.py

```diff
@@ -1,1364 +1,1341 @@
-                r"but got empty list"
-            ),
-        ) as exception_context:
-            search_versions("run_id IN ()")
-        assert exception_context.value.error_code == ErrorCode.Name(INVALID_PARAMETER_VALUE)
-
-        # search using an ill-formed IN operator correctly throws exception
-        with pytest.raises(
-            MlflowException, match=r"Invalid clause\(s\) in filter string"
-        ) as exception_context:
-            search_versions("run_id IN (")
-        assert exception_context.value.error_code == ErrorCode.Name(INVALID_PARAMETER_VALUE)
+    new_name = model_name + "!!!"
+    assert model_name != new_name
+    store.rename_registered_model(model_name, new_name)
+    assert store.get_registered_model(new_name).name == new_name
 
-        with pytest.raises(
-            MlflowException, match=r"Invalid clause\(s\) in filter string"
-        ) as exception_context:
-            search_versions("run_id IN")
-        assert exception_context.value.error_code == ErrorCode.Name(INVALID_PARAMETER_VALUE)
 
-        with pytest.raises(
-            MlflowException, match=r"Invalid clause\(s\) in filter string"
-        ) as exception_context:
-            search_versions("name LIKE")
-        assert exception_context.value.error_code == ErrorCode.Name(INVALID_PARAMETER_VALUE)
+def _extract_names(registered_models):
+    return [rm.name for rm in registered_models]
 
-        with pytest.raises(
-            MlflowException,
-            match=(
-                r"While parsing a list in the query, "
-                r"expected a non-empty list of string values, "
-                r"but got ill-formed list"
-            ),
-        ) as exception_context:
-            search_versions("run_id IN (,)")
-        assert exception_context.value.error_code == ErrorCode.Name(INVALID_PARAMETER_VALUE)
 
-        with pytest.raises(
-            MlflowException,
-            match=(
-                r"While parsing a list in the query, "
-                r"expected a non-empty list of string values, "
-                r"but got ill-formed list"
-            ),
-        ) as exception_context:
-            search_versions("run_id IN ('runid1',,'runid2')")
-        assert exception_context.value.error_code == ErrorCode.Name(INVALID_PARAMETER_VALUE)
-
-        # delete mv4. search should not return version 4
-        fs.delete_model_version(name=mv4.name, version=mv4.version)
-        assert set(search_versions("")) == {1, 2, 3}
-
-        assert set(search_versions(None)) == {1, 2, 3}
-
-        assert set(search_versions("name='%s'" % name)) == {1, 2, 3}
-
-        fs.transition_model_version_stage(
-            name=mv1.name, version=mv1.version, stage="production", archive_existing_versions=False
-        )
+def test_delete_registered_model(store, registered_models_names, rm_data):
+    model_name = registered_models_names[random_int(0, len(registered_models_names) - 1)]
```

Comment:
Why the random sampling here?

Reply:
I don't know. I kept it from the previous version.

Reply:
Yep, please simplify!